x/transfermiddleware/keeper/relay.go

```diff
@@ -43,7 +43,7 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 	voucherDenom := denomTrace.IBCDenom()
 	voucher := sdk.NewCoin(voucherDenom, transferAmount)
 
-	paraTokenInfo := k.GetParachainIBCTokenInfoByAssetID(ctx, data.Denom)
```

Comment:
It is better to move all logic from `ParachainIBCTokenInfoByAssetID` into old func `GetParachainIBCTokenInfoByAssetID` and do not introduce any changes on the caller side.

Reply:
rollback changes at https://github.com/ComposableFi/composable-cosmos/pull/408/commits/97af6caa1abfb3df7c0c062902f7d7d865af2123